src/libraries/System.Diagnostics.StackTrace/tests/StackTraceTests.cs

```diff
@@ -312,6 +312,14 @@ public void ToString_NullFrame_ThrowsNullReferenceException()
             Assert.Equal(Environment.NewLine, stackTrace.ToString());
         }
 
+        [Fact]
+        public unsafe void ToString_FunctionPointerSignature()
+        {
+            // This is sepate from ToString_Invoke_ReturnsExpected since unsafe cannot be used for iterators
+            var stackTrace = FunctionPointerParameter(null);
```

Comment:
Usage of explicit type (for reference: [Coding Guidelines](https://github.com/dotnet/runtime/blob/25d999b5a6711fc6d2ffc42fa38a58ec865a7232/docs/coding-guidelines/coding-style.md?plain=1#L26)))

```suggestion
            StackTrace stackTrace = FunctionPointerParameter(null);
```

Reply:
I see `var` style used a lot in this file. I would say most of the time.